src/debugger/Debugger.h

```diff
@@ -188,6 +189,7 @@ class Debugger : public gc {
 
     virtual bool processEvents(ExecutionState* state, Optional<ByteCodeBlock*> byteCodeBlock, bool isBlockingRequest = true) = 0;
 
+    bool exitClient = false;
```

Comment:
There is a virtual function to get this value, so it should be only in the remote debugger (I get you need the message because if the code runs too fast, the debugger has no chance to notify the server).